tools/monorepo-utils/src/slack/commands/slack/slack-message.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+
+/**
+ * Internal dependencies
+ */
+import { Logger } from '../../../core/logger';
+import { requestAsync } from '../../../core/util';
+
+type SlackResponse = {
+	ok: boolean;
+	error?: string;
+};
+
+export const slackMessageCommand = new Command( 'message' )
+	.description( 'Send a plain-text message to a slack channel' )
+	.argument(
+		'<token>',
+		'Slack authentication token bearing required scopes.'
+	)
+	.argument( '<channel>', 'Slack channel to send the message to.' )
```

Comment:
Should this be an array of channels?

Reply:
🤔 I don't have a strong reason not to support a comma separated string of channels so happy to do this.

Reply:
@psealock I used variadic arg for channels to resolve this https://github.com/woocommerce/woocommerce/pull/38185/commits/c86a95f07c907c33baf35ffdab744a5433cef2a9

Note that making channel last arg requires reshuffling args. I've updated the test instructions.